releasenotes/notes/38713.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: release-notes/v2
+kind: bug-fix
+area: traffic-management
+issue:
+  - 38713
```

Comment:
Also point at the issue, https://github.com/istio/istio/issues/38678.

Reply:
good catch, file re-named as well to match the issue number